### PR TITLE
Slightly Expands 3 Rooms of Cyberiad's AI sat, Tidies Furnishings. Cleans up Vents and Scrubber pipes. Improves Wallbump Placement.

### PR DESCRIPTION
**Pull request by Fordoxia**
## What Does This PR Do
Reduces the number of overlapping wallbumps.

The vent and scrubber pipes have been untangled.

Makes the foyer and maintenance rooms slightly larger and rearranges the furniture within.

The O2 locker in the transit tube area has been placed into the alcove between the tube exit and the window.

Moves the portable scrubbers into the maintenance halls.

Removes two racks with crowbars and welding masks. There's already a set of toolboxes and a mask in the east maintenance room.

All metal tables have been replaced with reinforced tables.

Two new tables added to the AI sat antechamber.

Relocated the Red Phone from the foyer to the antechamber.

## Why It's Good For The Game
Nicer distribution of wallbumps.

The west maint room was a cluttered to hell. Using the maint tunnels to store the portable scrubbers has freed up a lot of space, and means that a single meteor is far less likely to destroy both of them at once.

The atmos pipes and devices are much neater now.

Most light switches are now located by the doors so you can easily flick them on when you walk in.

The O2 locker in the transit tube area now gets in the way less by being located in the convenient alcove.

## Images of changes
image
image

## Testing
Visited and did a tour of the sat.

## Changelog
:cl:
tweak: Cyberiad AI Sat - Foyer and side maint rooms are slightly larger.
tweak: Cyberiad AI Sat - Furniture has been rearranged.
tweak: Cyberiad AI Sat - Removed some racks of crowbars.
tweak: Cyberiad AI Sat - Atmos pipes have been rearranged.
/:cl:

### CODE REVIEW

**Review by Burzah — CHANGES_REQUESTED** [review]
The PR description and what it does needs to be updated properly. Enough is enough with this. I want to see what the PR does from the title, and from the description. The Change log also needs to be updated properly for all the changes you have made. All the changes need to be documented properly. Leave the memes for coding chat and off your PRs. We want to know what the changes are without having to read through and decipher it.

**Review by FunnyMan3595 — CHANGES_REQUESTED** [review]

File: _maps/map_files/stations/boxstation.dmm [hunk 1]
```diff
@@ -39584,7 +39611,7 @@
 	d2 = 4;
 	icon_state = "1-4"
 	},
-/obj/machinery/door/window/classic/reversed{
+/obj/machinery/door/window/reinforced/reversed{
```
[the remark this answers is not in the document]
Comment (DGamerL) [reply _maps/map_files/stations/boxstation.dmm:39614]: (It doesn't)

[the remark this answers is not in the document]
Comment (Fordoxia) [reply _maps/map_files/stations/boxstation.dmm:39614]: Change made.

**Review by warriorstar-orion — APPROVED** [review]
LGTM

**Review by Burzah — APPROVED** [review]
Lgtm

**Review by FunnyMan3595 — APPROVED** [review]

**Pull request merged**